server/models/meshery_meshmodels_api_response.go

```diff
@@ -45,6 +46,14 @@ type MeshmodelRelationshipsAPIResponse struct {
 	Relationships []entity.Entity `json:"relationships"`
 }
 
+// API response model for meshmodel relationships view API
+type MeshmodelRelationshipsViewAPIResponse struct {
```

Comment:
No need to create a struct thier should be one already in meshery.
Regarding entity.Entity look up the code under directory 
`/server/handlers/component_generation.go` you would some code regarding component entity and would guide you how to fetch the response properly.

Reply:
> No need to create a struct thier should be one already in meshery. Regarding entity.Entity look up the code under directory `/server/handlers/component_generation.go` you would some code regarding component entity and would guide you how to fetch the response properly.

**Regarding entity.Entity look up the code under directory
/server/handlers/component_generation.go you would some code regarding component entity and would guide you how to fetch the response properly.**
But the issue here is that the entity.Entity type doesn't contain all the fields which the api returns
below are the fields which api returns and I found that []v1alpha2.RelationshipDefinition is there in meshery.
![image](https://github.com/user-attachments/assets/b017668e-16e4-4a12-80f0-32960efd8924)